### DISCUSSION TITLE
Contributing with code

### DISCUSSION THREAD

**Discussion by LeifParr** [petrobras/ross#847] [General]
@raphaeltimbo : We have started using github at work for our own code. So it seems to me that I may soon be ready to start contributing on a small scale for fixing small stuff in the ross code. Do I have to be allowed in somehow, or can I follow contribution.rst instructions, fork code, and get going?

**Comment by raphaeltimbo** [comment]
@LeifParr, good to hear that! 

> Do I have to be allowed in somehow, or can I follow contribution.rst instructions, fork code, and get going?

You can follow the contribution.rst. If you fork the project, you can open PRs from your fork to the ROSS repository.
Let me know if you have any doubts regarding the instructions in that file so that I can clarify and maybe improve the instructions.
